platform/ios/src/MGLMapboxEvents.m

```diff
@@ -12,8 +12,8 @@
 
 #include <sys/sysctl.h>
 
-static const NSUInteger version = 1;
```

Comment:
Instrumentile doesn't have control over userAgent, so it sends `version` -- we might be able to drop it for mobile events by checking the userAgent, but that's why it's around.